Form-Controls/index.html

```diff
@@ -1,27 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Form exercise</title>
+  <meta name="description"
+    content="The form collects client details about their purchase, item characteristics, and delivery information, including the address for shipping." />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="styles.css" />
 </head>
+
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>ZARA STORES</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <div class="container">
+      <form id="form" class="form">
+        <h2>Please choose the size and color of your  T-shirt</h2>
+        <div class="form-control">
+          <label for="usarname" >User</label>
+          <input type="text" id="username" name="username" placeholder="Enter Username" minlength="2" required />
+        </div>
+        <div class="form-control">
+          <label for="email" class="email">Email</label>
+          <input type="email" placeholder="Enter your email" pattern="[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*" required />
```

Comment:
Hi @Grajales-K , 
Did you intentionally omit the "id" attribute for email input element.

Reply:
thanks for this, @mrvicthor  I did not see it before, I already added thanks.